tests/integration/llm/prepare.py

```diff
@@ -129,11 +135,23 @@ def build_sd_handler_model(model):
     write_properties(options)
 
 
+def build_ft_raw_model(model):
+    if model not in ft_model_list:
+        raise ValueError(
+            f"{model} is not one of the supporting handler {list(ft_model_list.keys())}"
+        )
+    options = ft_model_list[model]
+    options["engine"] = "DeepSpeed"
```

Comment:
This can now be `FasterTransformer` if you rebase from master

Reply:
looks like this change is not complete, ran into this error - 
```
Model Store: /opt/ml/model
Initial Models: test=file:/opt/ml/model/test/
Initial Workflows: N/A
Netty threads: 0
Maximum Request Size: 67108864
[INFO ] 2023-02-10 00:01:12 ModelServer - Initializing model: test=file:/opt/ml/model/test/
[INFO ] 2023-02-10 00:01:15 ModelServer - Model server stopped.
Error:  2023-02-10 00:01:15 ModelServer - Invalid configuration: Deep learning engine not found: FasterTransformer
```

so keeping it DeepSpeed for now.

Reply:
docker container is not updated with latest build of djl-serving i guess, Will change it in a subsequent PR.